commit 6d583d8fcc66e8e5d4f6abe3253deeeb25331199 (refs/changes/27/927/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-11-21T22:39:16+00:00 (2 years, 11 months ago)
    
    MORAY-350 Moray throws ReferenceError: InternalError is not defined
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/test/sql.test.js b/test/sql.test.js
index c94c051..1a844fa 100644
--- a/test/sql.test.js
+++ b/test/sql.test.js
@@ -5,12 +5,13 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2016, Joyent, Inc.
  */
 
 var tape = require('tape');
 var util = require('util');
 var uuid = require('libuuid').create;
+var VError = require('verror');
 
 var helper = require('./helper.js');
 
@@ -85,7 +86,9 @@ test('sql - fail', function (t) {
     sql = 'BOGUS QUERY;';
     q = c.sql(sql, [], {});
     q.once('error', function (err) {
-        t.ok(err);
+        t.ok(err, 'error returned');
+        t.ok(VError.hasCauseWithName(err, 'InternalError'),
+            'InternalError returned');
         t.end();
     });
 });
